Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -0,0 +1,647 @@
+  API Key Pairs can be requested by emailing support@crowdstrike.com.
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: url
+  defaultvalue: https://falconapi.crowdstrike.com/
+  type: 0
+  required: true
+- display: API ID
+  name: id
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: useproxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Allow self-signed SSL certificates
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    var serverUrl = params.url;
+    if (serverUrl[serverUrl.length - 1] !== '/') {
+        serverUrl += '/';
+    }
+
+    var doReq = function(method, path, query, body) {
+        var result = http(
+            serverUrl + path + encodeToURLQuery(query),
+            {
+                Headers: {
+                    'Content-Type': ['application/json'],
+                    'Accept': ['application/json']
+                },
+                Username: params.id,
+                Password: params.key,
+                Method: method,
+                Body: body ? JSON.stringify(body) : ''
+            },
+            params.insecure,
+            params.useproxy
+        );
+        if (result.StatusCode < 200 && result.StatusCode > 299) {
+            throw 'Failed to perform request ' + path + ', request status code: ' + result.StatusCode + ', body: ' + result.Body;
+        }
+        if (result.Body === '') {
+            throw 'No content recieved.';
+        }
+        var obj;
+        try {
+            obj = JSON.parse(result.Body);
+        } catch (ex) {
+            throw 'Error parsing reply - ' + result.Body + ' - ' + ex;
+        }
+        if (obj.errors && obj.errors.length > 0) {
+            throw obj.errors.join(', ');
+        }
+        return {body: result.Body, obj: obj, statusCode: result.StatusCode};
+    };
+
+    var dateToEpoch = function(d) {
+        return d ? new Date(d).getTime() : null;
+    };
+
+    var uploadIOC = function() {
+        var res = doReq('POST', 'indicators/entities/iocs/v1', {}, [args]);
+        return {Type: entryTypes.note, Contents: res.body, ContentsFormat: formats.json, HumanReadable: 'IOC [' + args.value + '] uploaded'};
+    };
+
+    var getIOC = function() {
+        var res = doReq('GET', 'indicators/entities/iocs/v1', {ids: args.type + ':' + args.value});
+        var md = '### Falcon Host custom IOC retrieval\n';
+        if (res.obj.resources) {
+            for (var i=0; i<res.obj.resources.length; i++) {
+                md += objToList(res.obj.resources[i], res.obj.resources[i].value) + '\n';
+            }
+        } else {
+            md += 'No result found';
+        }
+        return {Type: entryTypes.note, Contents: res.body, ContentsFormat: formats.json, HumanReadable: md};
```

Comment:
Not sure what was doen in this  case.
But context can be placed via the output object in the YML